cf_xarray/parametric.py

```diff
@@ -0,0 +1,691 @@
+import inspect
+import sys
+
+import numpy as np
+import xarray as xr
+
+ocean_stdname_map = {
+    "altitude": {
+        "zlev": "altitude",
+        "eta": "sea_surface_height_above_geoid",
+        "depth": "sea_floor_depth_below_geoid",
+    },
+    "height_above_geopotential_datum": {
+        "zlev": "height_above_geopotential_datum",
+        "eta": "sea_surface_height_above_ geopotential_datum",
```

Comment:
```suggestion
        "eta": "sea_surface_height_above_geopotential_datum",
```

many of these errors below